requirements/edx/github.txt

```diff
@@ -45,7 +45,7 @@
 
 # Third-party:
 git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline==1.5.3
-git+https://github.com/edx/django-wiki.git@v0.0.7#egg=django-wiki==0.0.7
+git+https://github.com/edx/django-wiki.git@ff4cd9b6e139d76ad6f120b48eabed12123bbfbf#egg=django-wiki==0.0
```

Comment:
Don't forget to bump the version to "0.0.8", when you're done with merging the wiki PR.